tools/kmod/sof_remove.sh

```diff
@@ -58,6 +65,9 @@ remove_module snd_sof_probes
 remove_module snd_sof_intel_client
 remove_module snd_sof_client
 
+#-------------------------------------------
+# Codec driver
+#-------------------------------------------
```

Comment:
this should be two lines down. This part  should be 'Remaining core SOF parts"

Reply:
I moved down by two lines,

I believe this is applicable to below section(last block)
>This part should be 'Remaining core SOF parts"